utils/constants/graph-constants.js

```diff
@@ -0,0 +1,83 @@
+export const graphOptions = {
```

Comment:
I would like to rename this constant in uppercase, i.e. `GRAPH_OPTIONS` instead of `graphOptions`, that will help us to identify a constant.

Reply:
Done